pom.xml

```diff
@@ -214,6 +214,46 @@
 
     <dependencyManagement>
         <dependencies>
+            <?SORTPOM IGNORE?>
+            <!--
+                Temporary workaround to settle opentelemetry on 1.37.0 version.
```

Comment:
How temporary? 

BTW, if the order the declarations appear in the dependencyManagement section matters (ouch!), then it's improper to try to sort them. Should we get rid of sorting altogether?

Reply:
until the `libraries-bom` are updated (the chain of BOM imports is: libraries-bom -> first-party-dependencies bom -> gapic-generator-java bom -> opentelemetry bom)

Reply:
The order of the declarations matter for BOMs which imports their dependencies into the current scope and first import wins.

Reply:
I think that the new libraries-bom is coming soon and it should contain new opentelemetry import that doesn't downgrade our own requirement.

Reply:
https://github.com/googleapis/java-cloud-bom/releases/tag/v26.38.0 was tagged an hour ago

Reply:
Just in time. Dropped.